docs/configuration.rst

```diff
@@ -947,6 +947,23 @@ Description
     Download full-sized original images if available.
 
 
+extractor.fanbox.videos
+-----------------------
+Type
+    ``bool`` or ``string``
+Default
+    ``true``
+Description
+    Control behavior on videos embedded from external sites.
+    Recognizes embeds from YouTube, Vimeo and SoundCloud.
+
+    * ``true``: Extract video URLs (videos are not downloaded, as
+      galley-dl does not support these sites natively)
+    * ``"ytdl"``: Download videos and let `youtube-dl`_ handle all of
+      video extraction and download
+    * ``false``: Ignore videos
```

Comment:
I think this option should be simplified to just
* `true`: Download embedded media with youtube-dl (what is currently happening for `"ytdl"`)
* `false`: Ignore videos

with just "Download embedded media from  YouTube, Vimeo, and SoundCloud with youtube-dl" as description.
The current `true` would download the HTML content of those external sites and I don't think anyone would want that.

Reply:
Hmm, `true` shouldn't download html, instead the links show up in the unsupported URLs (the `--write-unsupported` flag),
though I can remove it, I don't particularly mind either way.

Reply:
You are right, they would land in the "unsupported" file. I didn't realize it's using`Message.Queue`.
But that wouldn't use youtube-dl to download them, either. It's `Message.Url` to download and `Message.Queue` to potentially spawn a new extractor.

I would still only have `true` or `false` as options (because its easier).
It'd need to use `Message.Url` for `ytdl:…` URIs if you want to keep all three.